10_install_server/00_install_intro.txt

```diff
@@ -10,22 +10,27 @@ Architecture chapter.
 
 Ideally, this machine should have Internet access, but this is not a strict requirement.
 
-As Rudder datas can grow really fast depending on your number of managed nodes and number of rules, it is advised to separate partitions to prevent your /var getting full and break your system.
+As Rudder datas can grow really fast depending on your number of managed nodes and number of rules, it is advised to separate partitions to prevent /var from getting full and break your system.
 Special attention should be given to:
 
 ======
 
 /var/lib/pgsql::
-Or wherever is located your postgresql database.
-Can grow by several GB per day.
-Please see the <<_database_maintenance, database maintenance>> chapter for more details about this
+( OS dependent ).
+Please see the <<_database_maintenance, database maintenance>> chapter for more details about the
+PostgreSQL database size estimation.
 
 /var/rudder::
-Contains most of your server information, LDAP database, etc..
-Slower growth over time.
+Contains most of your server information, the configuration-repository, LDAP database, etc..
+Rudder application-related files should stay under 1GB, but the size of the configuration-repository will
+depend of the amount of data you store in it, especially in the shared-files folder (files that will get
+distributed to the agents using the "Download a file for the shared folder" Technique).
 
 /var/log/rudder:: 
-Reports logs can easily grow to 1.5GB per day.
+Report logs (/var/log/rudder/reports) size will depend on the amount of nodes you manage (should stabilize
+around 1GB per 20 nodes).
```

Comment:
This is no random number but an estimation based on what we have on our own production, consolidated by tests on diff. machines. Remember that we aren't talking about Postgres reports here, but plain text ones, that are rotated weekly by logrotate and not stored using postgres engine (with a little overhead).